test/integration/framework/apiserver.go

```diff
@@ -96,31 +133,22 @@ func init() {
 	apiextensionsinstall.Install(internalScheme)
 }
 
+// patchCRDConversion overrides the conversion configuration of the CRDs that
+// are loaded in to the integration test API server,
+// configuring the conversion to be handled by the local webhook server.
 func patchCRDConversion(crds []apiextensionsv1.CustomResourceDefinition, url string, caPEM []byte) {
-	for _, crd := range crds {
-		for i := range crd.Spec.Versions {
-			crd.Spec.Versions[i].Served = true
-		}
-		if crd.Spec.Conversion == nil {
-			continue
-		}
-		if crd.Spec.Conversion.Webhook == nil {
-			continue
-		}
-		if crd.Spec.Conversion.Webhook.ClientConfig == nil {
-			continue
-		}
-		if crd.Spec.Conversion.Webhook.ClientConfig.Service == nil {
-			continue
-		}
-		path := ""
-		if crd.Spec.Conversion.Webhook.ClientConfig.Service.Path != nil {
-			path = *crd.Spec.Conversion.Webhook.ClientConfig.Service.Path
+	for i := range crds {
+		url := fmt.Sprintf("%s%s", url, "/convert")
+		crds[i].Spec.Conversion = &apiextensionsv1.CustomResourceConversion{
+			Strategy: apiextensionsv1.WebhookConverter,
+			Webhook: &apiextensionsv1.WebhookConversion{
+				ClientConfig: &apiextensionsv1.WebhookClientConfig{
+					URL:      &url,
+					CABundle: caPEM,
+				},
+				ConversionReviewVersions: []string{"v1"},
+			},
```

Comment:
This function now unconditionally sets rather than adjusts conversion configuration, because the new test CRDs do not have any conversion configuration.